src/index.html

```diff
@@ -7,12 +8,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../src/block/index.scss"
     />
+    <!-- #endregion -->
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="product"
+    >
+      <img
+        class="product__card"
+        src="../src/images/imac.jpeg"
+        alt="APPLE A1419"
+      />
+
+      <h2 class="product__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div>
+        <p class="product__code">Product code: 195434</p>
+      </div>
```

Comment:
This div is redundant
```suggestion
      <p class="product__code">Product code: 195434</p>
```